dirty_cat/datasets/_ken_embeddings.py

```diff
@@ -24,6 +24,8 @@ def get_ken_embeddings(
     ----------
     types : str
         Types of embeddings to include. Write in lowercase.
+        For the full list of accepted types, see the `entity_detailed_types`
+        table on https://soda-inria.github.io/ken_embeddings/.
```

Comment:
I agree that being able to see all types in the browser would be great! However there are a lot of them, so I don't think it can fit in one page. Ideally there would be a search bar where the user can search for types present in YAGO3, or find the type of a given entity (e.g. a video game).

Reply:
OK, maybe we need to upload a table with two columns: type name and number of occurences to a google doc spreadsheet and share this?

Reply:
The upper limit for the number of rows in Google spreadsheet is 40 000

Reply:
This `types` parameter was meant to facilitate the needs of the user as it already filters the types containing the input string. 
So if you put "music" you will get all types with this string (e.g. "wikicat_musician_from_france", "wikicat_music_label" etc.)
Going directly for the exact type name (e.g. "wikicat_rock_music_bands") seems a bit unnecessary.

But I agree it might be useful to have the most common types. We have 114 509 types in yago3 2022. But I guess it would be best to take the top 40 000 types with their number of occurrences, put it into a google doc spreadsheet and share it. There are 84 entities in the 40 000th type, so I guess it's ok to leave out the other ones.

Reply:
OK, so it should be better described: that it is a substring and that it will be matched (the paragraph that you have above is actually quite useful.

I wonder if we should not provide a helper function to help people tune this substring: "get_ken_matching_types" that would return all the types that have the corresponding substring, and expose it to the users. I guess that we can do this in a second time, after merging this PR.